src/js/helpers/graph/declarationHelper.js

```diff
@@ -14,8 +14,10 @@
  * limitations under the License.
  */
 
-// eslint-disable-next-line import/no-cycle
 import GraphLayers from './graphLayers';
+import {
+  isReference, hasTransitions, hasPreDefinedTransitions, hasTransition, isComposite, getReference,
```

Comment:
nit. I would spread it into multiline - I find it more readable.

Reply:
Everywhere else we have such things formatted in one line, so it's probably better to leave it like this for consistency

Reply:
Of course, I agree.